tests/comments/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -106,17 +106,100 @@ class Foo
         // there should be no docblock here
         $this->title = $title;
     }
+
+    /**
```

Comment:
Let's check `isDoc` comment (maybe use regex or package), if yes - pretty output, if no - print as is :+1: